libs/server/Resp/Objects/SortedSetCommands.cs

```diff
@@ -193,6 +193,53 @@ private unsafe bool SortedSetRange<TGarnetApi>(RespCommand command, ref TGarnetA
             return true;
         }
 
+        /// <summary>
+        /// ZREVRANGEBYLEX 
+        /// </summary>
+        /// <typeparam name="TGarnetApi"></typeparam>
+        /// <param name="storageApi"></param>
+        /// <returns></returns>
+        private bool ListRangeByLength<TGarnetApi>(ref TGarnetApi storageApi)
+             where TGarnetApi : IGarnetApi
+        {
+            if (parseState.Count is not (3 or 6))
+            {
+                return AbortWithWrongNumberOfArguments(nameof(RespCommand.ZREVRANGEBYLEX));
+            }
+
+
+            var sbKey = parseState.GetArgSliceByRef(0);
+            var start = parseState.GetArgSliceByRef(1);
+            var stop = parseState.GetArgSliceByRef(2);
+            var byLenOption = ArgSlice.FromPinnedSpan(CmdStrings.BYLEX);
+            var revOption = ArgSlice.FromPinnedSpan(CmdStrings.REV);
+
+            // ZREVRANGEBYLEX key start stop
+            if (parseState.Count == 3)
+            {
+                parseState.InitializeWithArguments(sbKey, start, stop, byLenOption, revOption);
```

Comment:
Again, no need to initialize a new parse state, actually no need for this method at all, just create a new SortedSetOperation - ZREVRANGEBYLEX and call SortedSetRange directy, then handle that specific op in SortedSetObjectImpl.

Reply:
Done